src/map/battle.c

```diff
@@ -1407,6 +1407,17 @@ int64 battle_calc_damage(struct block_list *src,struct block_list *bl,struct Dam
 				status_change_end(bl, SC_KYRIE, INVALID_TIMER);
 		}
 
+		if ((sce = sc->data[SC_P_ALTER]) && damage > 0) {
+			clif_specialeffect(bl, 336, AREA);
+			sce->val3 -= (int)cap_value(damage, INT_MIN, INT_MAX);
```

Comment:
Maybe `sce->val3 = (int)cap_value(sce->val3 - damage, INT_MIN, INT_MAX);`
so the followed if-else isn't needed?